charts/socat-tunneller/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for socat-tunneller
 name: socat-tunneller
-version: 0.1.5
+version: 0.1.6
```

Comment:
I think this change is a new feature, so justifies this bumping to `0.2.0`.

Reply:
ok 🎉